lightly/api/api_workflow_client.py

```diff
@@ -105,6 +108,7 @@ def __init__(
             self._dataset_id = dataset_id
         if embedding_id is not None:
             self.embedding_id = embedding_id
+        self.dataset_creator = dataset_creator
```

Comment:
Did you consider making the attribute private? In my eyes it should not be changed after the client is created.

Reply:
Good point, I changed it.